courses/machine_learning/deepdive2/structured/solutions/1_explore_babyweight.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>Remove the code comment directly above the function, since you already have the docstring.</p><p> </p><p>In the docstring:</p><ul><li>statstics -&gt; statistics</li><li>valid_columns_set -&gt; columns_set (see comment above)</li></ul><p> </p><p>In the function body:</p><ul><li>valid_columns_set -&gt; columns_set (see comment above)</li><li>pep8: Break the following line (it's too long):</li></ul><p> </p><p><code>&nbsp;&nbsp;df.plot(x=column_name, y="num_babies", logy=logy, kind="bar", figsize=(12, 5))</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done</p>